### ISSUE TITLE
[Arcs] Securing a Vox Crisis = Harm?

### ISSUE THREAD

**Issue by ledergamesmatt** [LederCards/cards#33]
does securing Terror strike counts as harming (for the purpose of the Truce Law)
[Terror Strike](https://cards.ledergames.com/card/ARCS-F621?q=terror%20strike)

- [ ] F620
- [ ] F621

**Comment by turmoilbyrd** [comment]
From https://discord.com/channels/476234833572397056/963112189802201108/1259608208801140850

**Comment by turmoilbyrd** [comment]
My current understanding is

the player securing Terror strike
- is not violating Truce (can do it)
- doesn't get trophies
- doesn't ransack the court or provoke outrage

**Comment by jdyearsley** [comment]
Correct, the Crisis effect should be resolved exactly as a Crisis, as @turmoilbyrd described. I'll add a FAQ to this effect.
